src/imdb.ts

```diff
@@ -28,6 +28,7 @@ const omdbapi = "https://www.omdbapi.com/";
 export interface MovieOpts {
     apiKey: string;
     timeout?: number;
+    plot?: string;
```

Comment:
Can this be made into an enum? I'd be much happier supporting that since the set of possible options for this are limited.